cypress/integration/11_admin_dataset_file_upload.spec.js

```diff
@@ -82,8 +85,8 @@ context('Admin dataset file upload', () => {
 
     before(() => {
       const selectorDist = '#edit-field-json-metadata-0-value-distribution-distribution-0-distribution-downloadurl-upload'
-
```

Comment:
Could we add this const out into the context since it's used in two separate places in the context?